docassemble/AssemblyLine/al_document.py

```diff
@@ -931,23 +942,50 @@ def download_list_html(self, key:str='final', format:str='pdf', view:bool=True,
     filename_root = os.path.splitext(str(self.filename))[0]
     if len(enabled_docs) > 1 and include_zip:
       zip = self.as_zip(key=key)
-      zip_button = action_button_html( zip.url_for(attachment=False, display_filename = filename_root + ".zip"), label=zip_label, icon=zip_icon, color="primary", size="md", classname='al_zip' )
+      zip_button = action_button_html(
+          zip.url_for(attachment=False, display_filename = filename_root + ".zip"),
+          label=zip_label, icon=zip_icon, color="primary", size="md", classname='al_zip' )
       html += table_row( zip.title, zip_button)
-      
+
     html += '\n</table>'
 
     # Discuss: Do we want a table with the ability to have a merged pdf row?
     return html
 
   def download_html(self, key:str ='final', format:str ='pdf',
-                    view:bool=True, refresh:bool=True) -> str:
+                    view:bool=True, refresh:bool=True,
+                    view_label:str='View', view_icon:str='eye',
+                    download_label:str="Download", download_icon:str="download") -> str:
     """
     Returns an HTML string of a table to display all the docs
     combined into one pdf with 'view' and 'download' buttons.
     """
-    the_file = self.as_pdf(key=key)
+    if format == 'docx':
+      the_file = self.as_docx(key=key)
+    else:
+      the_file = self.as_pdf(key=key)
```

Comment:
Is this the one substantive change? Trying to figure out the impact here mixed w/ line breaks.

Reply:
Sorry, I did kinda go crazy on the line breaks.

And yes, it is substantive. Before, nothing depended on `format`, and now we're trying to preserve pdf vs docx when it's passed in.